### ISSUE TITLE
Make media license optional

### ISSUE THREAD

**Issue by ClobberXD** [minetest/contentdb#150]
Package authors are forced to select a media license, as there's no "None" (or similar) option. This makes things tricky for packages that don't contain any media whatsoever, like APIs and frameworks. See a live example [here](https://content.minetest.net/packages/ANAND/callbacks_api)

**Comment by Wuzzy2** [comment]
Be careful with that one. People are going to assume that it will be OK to deliberately use no license even if the mod does have media. Which, IMHO, it is not!
Note that “no license” will usually default to “fully copyrighted”, which is unacceptable for a free community.

Better reword it to something like “Does not apply / no media included”.

**Comment by Wuzzy2** [comment]
But I agree this is a major bug in the Content DB.

In the meanwhile, here's a workaround: Just choose the same license as for the code.
